src/main/scala/lms/transformation/distributed_tensor/distributedTensorBase.scala

```diff
@@ -110,6 +120,11 @@ trait FixedSizeDistributedTensorBaseTypeLess {
       }
     }
 
+    def op_type = gc.get(x.asInstanceOf[Backend.Sym]) match {
+      case Some(Node(_, s, _, _)) if s.startsWith("tensor") => s
+      case a => throw new Exception(s"cannot find node $a")
+    }
+
     def shape_size: Seq[Int] = tensor_type.shape.map(_.size)
```

Comment:
Suggestion: `op_type` -> `opType`, `share_size` to `shapeSize`, etc.

Reply:
good point.